### ISSUE TITLE
Overlapping Tags Use the same Format

### ISSUE THREAD

**Issue by stealthyninja671** [mdelobelle/obsidian_supercharged_links#134]
I noticed that the program can't seem to differentiate between a the tags #group and #groupmeeting, I have formatting applied to the tag #group, and it automatically applies to the second one

I could workaround this by applying a blank style to the #groupmeeting tag, but this seemed like a potential issue down the line.

**Comment by drhayes** [comment]
Yup, came here to post this one as well: `#person` vs. `#personal-mythology` both get the same rule applied, but I only wanted it on `#person`.

**Comment by HEmile** [comment]
Yeah this is true, but also somewhat by design when you have hierarchical tags. I think I know a workaround, though.

**Comment by drhayes** [comment]
I'm very new to Obsidian. Is there a way to do hierarchical tags besides using a `/` to split them? I'm also migrating from a different note system and converting tags. 😁

**Comment by Gewerd-Strauss** [comment]
Just noticed this as well, and it frankly confuses me. 
`#log`, and `#knowledge/biology` are certainly different things, and I don't want my notes on molecular biology to be tagged with my log-icon. I would appreciate a fix for this.
